share/completions/xrandr.fish

```diff
@@ -39,6 +39,32 @@ complete -c xrandr -s o -l orientation -d 'Specify the orientation of the screen
 complete -c xrandr -s x -d 'Reflect across the X axis'
 complete -c xrandr -s y -d 'Reflect across the Y axis'
 
+# Version > 1.5
+if test $ver[1] -gt 1; or test "$ver[2]" -ge 5
+    complete -c xrandr -l listmonitors -d 'Print all defined monitors'
+    complete -c xrandr -l listactivemonitors -d 'Print all active monitors'
+    complete -c xrandr -l setmonitor -d 'Define new monitor' -x
+    complete -c xrandr -l delmonitor -d 'Delete monitor' -x
+end
+
+# Version > 1.4
+if test $ver[1] -gt 1; or test "$ver[2]" -ge 4
+    complete -c xrandr -l listproviders -d 'Print all available providers'
+    complete -c xrandr -l setprovideroutputsource -d 'Set source for a given provider'
+    complete -c xrandr -l setprovideroffloadsink  -d 'Set provider for a given sink'
+end
+
+# Version > 1.3
```

Comment:
Version 1.5 was released in 2016, 1.4. in 201**3**. There is no need to check that far back, especially when it just comes to offering options that might not work.

Reply:
If you wanted to call `xrandr` in the completion script, and that call required a version, I might be convinced to keep the check for 1.5.

But if it's just about offering options? Nah, ditch them.

Reply:
In another project someone on Ubuntu 18.04 LTS (IIRC) still had an older version by default. We can just get rid of all version checks since this is probably one of a couple completions that actually check the version.

Reply:
> Nah, ditch them.

Sayonara

Reply:
I think you might be mistaken, since Ubuntu **16**.04 (i.e. the oldest still supported version) has 1.5: https://packages.ubuntu.com/search?suite=xenial&searchon=names&keywords=xrandr

Ubuntu *14*.04 might not have, but we don't even build fish 3.1.2 for it in our PPA.

And even *iff* someone ran this on a distribution that old, the failure mode is quite benign - try an option, see that it doesn't work, know you have to find a workaround (like, you know, upgrading to something not old enough to go to school).

Reply:
It was a different X utility, but your point still stands.